toolkit/offchain/src/d_param/tests.rs

```diff
@@ -17,19 +17,17 @@ mod mint_tx {
 	use super::*;
 	use cardano_serialization_lib::Transaction;
 
-	fn mint_ex_units() -> ExUnits {
+	fn ex_units() -> ExUnits {
 		ExUnits::new(&10000u32.into(), &200u32.into())
 	}
-	fn ex_units() -> ScriptExUnits {
-		ScriptExUnits::new().with_mint_ex_units(vec![mint_ex_units(), mint_ex_units()])
-	}
 	fn mint_d_param_tx() -> Transaction {
 		mint_d_param_token_tx(
 			&test_validator(),
 			&test_policy(),
 			&input_d_param(),
 			&test_tx_context(),
 			ex_units(),
+			ex_units(),
```

Comment:
Can you please use different ex_units() compared to what is above, so it can be check that they are attached to proper redeemers?